target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -201,6 +201,11 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:lan" "2:lan" "3:lan" "4:lan"
 		;;
+    tplink,tl-wr941hp-v1)
+		ucidef_set_interface_wan "eth1"
+		ucidef_add_switch "switch0" \
+			"0@eth0" "1:lan:4" "2:lan:3" "3:lan:2" "4:lan:1"
```

Comment:
Can this be merged with an existing case?

Reply:
Yes, it can. Done.